src/firmware/boards/legacy_robot_stm32f4/main.c

```diff
@@ -534,7 +535,8 @@ static void run_normal(void)
     FirmwareBall_t* ball =
         app_firmware_ball_create(dr_get_ball_position_x, dr_get_ball_position_y,
                                  dr_get_ball_velocity_x, dr_get_ball_velocity_y);
-    FirmwareWorld_t* world = app_firmware_world_create(robot, ball);
+    FirmwareWorld_t* world =
+        app_firmware_world_create(robot, ball, get_current_freertos_tick_time_seconds);
```

Comment:
interesting to use rtos time

Reply:
It was the easiest way to get time in the firmware :man_shrugging:. A more complete design might be adding in a new abstraction for setting time before the app layer is ticked (so that a single tick of the app layer has the same time throughout), but I was really trying to avoid changes that couldn't be tested without a robot.